build.props

```diff
@@ -4,11 +4,11 @@
     <EnlistmentRoot>$(MSBuildThisFileDirectory)</EnlistmentRoot>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)\obj</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)obj</BaseIntermediateOutputPath>

```

Comment:
Technically in the global build.props the trailing backslash is removed with a Trim().  In this context, it is always set to $(MSBuildThisFileDirectory) and will have a trailing backslash so the other properties are assuming it does.  I could have this build.props also trim it if you think it should.